profiling/simple-kernel-timer/kp_kernel_timer.cpp

```diff
@@ -50,25 +55,94 @@ void kokkosp_init_library(const int loadSeq, const uint64_t interfaceVer,
 }
 
 void kokkosp_finalize_library() {
-  double finishTime = seconds();
+  const char* kp_kernel_timer_json_raw = getenv("KP_KERNEL_TIMER_JSON");
+  const bool kp_kernel_timer_json =
+      strcmp(kp_kernel_timer_json_raw, "1") == 0 ||
+      strcmp(kp_kernel_timer_json_raw, "true") == 0 ||
+      strcmp(kp_kernel_timer_json_raw, "True") == 0;
+
+  double finishTime  = seconds();
+  double kernelTimes = 0;
+
+  char* mpi_rank = getenv("OMPI_COMM_WORLD_RANK");
```

Comment:
I would drop the MPI support from this PR.
This can be handled in a separate PR as needed.